src/floor/RequestSimulator.java

```diff
@@ -0,0 +1,91 @@
+package floor;
+
+import java.io.IOException;
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.SocketException;
+
+import messages.ElevatorRequestMessage;
+import messages.ElevatorRequestMessage.Direction;
+import messages.Message;
+
+public class RequestSimulator implements Runnable{
+	
+	DatagramSocket sendSocket;
+	DatagramPacket sendPacket;
+	
+	public RequestSimulator() {
+		try {
+			sendSocket = new DatagramSocket();
+		} catch (SocketException e) {
+			e.printStackTrace();
+		    System.exit(1);
+		}
+	}
+	
+	@Override
+	public void run() {
+		
+		try {
+			Thread.sleep(1000);
+		} catch (InterruptedException e) {
+			e.printStackTrace();
+		}
+		
+		ElevatorRequestMessage m = new ElevatorRequestMessage();
+		m.setDirection(Direction.UP);
+		m.setFloor(1);
+		byte[] data = Message.serialize(m);
+		//inform the floor subsystem
```

Comment:
We're supposed to expect that the message goes to the correct floor subsystem right? Not one big sub system?